ttnn/CMakeLists.txt

```diff
@@ -50,6 +50,7 @@ set(TTNN_SRCS
     ${CMAKE_CURRENT_SOURCE_DIR}/cpp/ttnn/operations/examples/example/device/example_device_operation.cpp
     ${CMAKE_CURRENT_SOURCE_DIR}/cpp/ttnn/operations/examples/example/device/single_core_program_factory.cpp
     ${CMAKE_CURRENT_SOURCE_DIR}/cpp/ttnn/operations/examples/example/device/multi_core_program_factory.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/cpp/ttnn/operations/experimental/argmax/argmax.cpp
```

Comment:
Why is it in experimental?

Reply:
https://github.com/tenstorrent/tt-metal/pull/10437#discussion_r1688542822
I have addressed @TT-BrianLiu comments.